tests/data/zyxelac_xmg3927.json

```diff
@@ -0,0 +1,3885 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.890",
+                    "sysDescr": "AC2800 (4x4 11ac + 3x3 11n) G.fast/VDSL 35b (Vplus) selectable IAD",
+                    "sysContact": "<private>",
+                    "version": "",
+                    "hardware": null,
```

Comment:
Could we also extract some of these fields? Hardware, version

Reply:
Unfortunately, nothing else is available in the data received from the device. Even "sysDescr" name (AC2800) is not the actual name of the device. It might be a rebranded device from another vendor.

Reply:
Boring!